tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -498,7 +512,8 @@ export class LineChartComponent
     this.setIsViewBoxOverridden(false);
     this.isViewBoxChanged = true;
     this.updateLineChart();
-    this.viewBoxChanged.emit(this.viewBox);
+    this.viewBoxChanged.emit(null);
```

Comment:
If I understand correctly, you have now fully implemented the redux pattern for view box. Should onViewBoxChanged() and viewBoxReset() therefore just both be single-line functions that only emit viewBoxChanged events? Could all the other code in these functions be deleted? And then, because of the redux pattern you're implemetned, the userViewBox Input will change and then updateLineChart() will be called as a side-effect of that?